src/erlcloud_sqs.erl

```diff
@@ -332,11 +347,28 @@ decode_message(Message) ->
        {md5_of_body, "MD5OfBody", text},
        {message_id, "MessageId", text},
        {receipt_handle, "ReceiptHandle", text},
-       {attributes, "Attribute", fun decode_msg_attributes/1}
+       {attributes, "Attribute", fun decode_msg_attributes/1},
+       {message_attributes, "MessageAttribute", fun decode_message_attributes/1}
       ],
       Message
      ).
 
+decode_message_attributes(Attributes) ->
+    [{decode_msg_attribute_name(Name), decode_message_attribute_value(DataType, StringValue)} ||
+        {Name, DataType, StringValue} <- decode_message_attribute(Attributes)].
+
+decode_message_attribute(Attributes) ->
+    [{erlcloud_xml:get_text("Name", Attr),
+      erlcloud_xml:get_text("Value/DataType", Attr),
+      erlcloud_xml:get_text("Value/StringValue", Attr)} || Attr <- Attributes].
+
+decode_message_attribute_value("Number", Value) ->
```

Comment:
Since [this logic](https://github.com/erlcloud/erlcloud/blob/81243b191550256c6452e35f7455d89b15759bac/src/erlcloud_ddb_streams.erl#L196) is already present elsewhere in the project to determine whether a string value represents a float or an integer I'll go ahead and use the same logic here.